indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -0,0 +1,296 @@
+            disabled={!allowFullDays}
+            label={label}
+          />
+        </Accordion.Title>
+      </Accordion>
+      {data
+        .map(item => camelizeKeys(item))
+        .map(({fullTitle, id}) => (
+          <dd className="grouped-fields" key={`session-block-checkbox-${id}`}>
+            <Checkbox
+              value={id}
+              onChange={handleValueChange}
+              checked={value.includes(id)}
+              label={fullTitle}
+              className={isExpandedClass}
+            />
+          </dd>
+        ))}
+    </>
+  );
+}
+
+SessionBlockHeader.propTypes = {
+  value: PropTypes.arrayOf(PropTypes.number).isRequired,
+  data: PropTypes.arrayOf(PropTypes.object).isRequired,
+  label: PropTypes.string.isRequired,
+  allowFullDays: PropTypes.bool.isRequired,
+  display: PropTypes.string.isRequired,
+  handleValueChange: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+export default function TimetableSessionsInput({
+  htmlId,
+  htmlName,
+  disabled,
+  isRequired,
+  allowFullDays,
+  display,
+  minimum,
+  maximum,
+}) {
+  const {eventId} = useSelector(getStaticData);
+  const {data: sessionData} = useIndicoAxios({url: timeTableURL({event_id: eventId})});
+
+  const minMsg = minimum > 0 ? `at least ${minimum}` : ``;
+  const and = minimum > 0 && maximum > 0 ? ` and ` : ``;
+  const maxMsg = maximum > 0 ? `no more than ${maximum}` : ``;
+
+  return (
+    <FinalField
+      id={htmlId}
+      name={htmlName}
+      component={TimetableSessionsComponent}
+      disabled={disabled}
+      required={isRequired}
+      allowFullDays={allowFullDays}
+      display={display}
+      minimum={minimum}
+      maximum={maximum}
+      sessionData={sessionData}
+      validate={value => {
+        if (value.length < minimum || value.length > maximum) {
+          return PluralTranslate.string(
+            'Please select {0}{1}{2} session block.',
+            'Please select {0}{1}{2} session blocks.',
+            maximum,
+            {0: minMsg, 1: and, 2: maxMsg}
+          );
+        }
+      }}
+    />
+  );
+}
+
+TimetableSessionsInput.propTypes = {
+  htmlId: PropTypes.string.isRequired,
+  htmlName: PropTypes.string.isRequired,
+  isRequired: PropTypes.bool.isRequired,
+  disabled: PropTypes.bool,
+  display: PropTypes.string,
+  allowFullDays: PropTypes.bool,
+  minimum: PropTypes.number,
+  maximum: PropTypes.number,
+};
+
+TimetableSessionsInput.defaultProps = {
+  disabled: false,
+  display: 'expanded',
+  allowFullDays: false,
+  minimum: 0,
+  maximum: 0,
+};
+
+export function TimetableSessionsSettings() {
+  return (
+    <>
+      <Form.Group inline>
+        <label>{Translate.string('Default display for sessions')}</label>
+        <FinalRadio name="display" label={Translate.string('Expanded.')} value="expanded" />
+        <FinalRadio name="display" label={Translate.string('Collapsed.')} value="collapsed" />
```

Comment:
```suggestion
        <FinalRadio name="display" label={Translate.string('Expanded')} value="expanded" />
        <FinalRadio name="display" label={Translate.string('Collapsed')} value="collapsed" />
```
And I'd add context to these two, otherwise it's hard to translate in gendered languages